### DISCUSSION TITLE
Envers tables not being generated but sql queries printed out to console output instead

### DISCUSSION THREAD

**Discussion by kbntk** [quarkusio/quarkus#36209] [Q&A]
Hi there,

I want to use hibernate envers in my quarkus application. However I stumble upon a problem when I annotate the classes with the '@Audited' annotation. The problem is that the audit tables are not being generated automatically but their `create table` statements are printed out in the console log. 

Could you please support me with resolving this problem?

The envers version I'm using is 3.4.1

**Comment by raphaelversiani** [comment]
Hi, I've found this answer below on stackoverflow. It worked for me. Hope it helps!

In your hibernate properties file, use this:

hibernate.hbm2ddl.auto=update

If you are using Spring's configuration to specify the value, then you would use:

spring.jpa.hibernate.ddl-auto=update
